<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>960</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Height map viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="image_display">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="image_viewer">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="opengl_display">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="DisplayGLWidget" name="openGL_viewer">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="grayscaling">
          <property name="text">
           <string>Grayscale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="standardColoring">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="HSVIng">
          <property name="text">
           <string>Color HSV</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="smoothing">
          <property name="text">
           <string>Smooth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bluring">
          <property name="text">
           <string>Blur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StreamAreaD8_Button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>StreamArea D8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StreamAreaSteepestButton">
          <property name="text">
           <string>StreamArea Steepest</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StreamPowerButton">
          <property name="text">
           <string>StreamPower</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="WetnessIndexButton">
          <property name="text">
           <string>Wetness Index</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RoadAction">
          <property name="text">
           <string>Road</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_image"/>
    <addaction name="separator"/>
    <addaction name="actionSave_image"/>
    <addaction name="actionSave_OBJ"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionImage_view"/>
    <addaction name="action3D_model"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionOpen_image">
   <property name="text">
    <string>Open image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionSave_image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_OBJ">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save OBJ</string>
   </property>
  </action>
  <action name="actionImage_view">
   <property name="text">
    <string>Image</string>
   </property>
  </action>
  <action name="action3D_model">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>3D model</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>displayglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
